app/helpers/games_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GamesHelper
```

Comment:
Pardon me for the nitpick, but I don't think we need this module just yet.

Reply:
File has been deleted in https://github.com/amykingOW/hangman/pull/1/commits/930179bcccc29b6a176a5b9061f971c6df0f24d1.